server/models.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable */
+// Written for Postgres
+const Image = require('../database/index_postgres');
+
+module.exports = {
+  restaurants: {
+    get: (params, callback) => {
+      Image.findAll({
+        where: {
+          userid: parseInt(params)
+        }
+      }).then(function (result) {
+        callback(result);
+      }).catch(function(err) {
+        callback(err)
+      });
+    },
+    post: (params, callback) => {
+      Image.create({
+        comment: null,
+        date: null,
+        imageurl: null,
+        userid: params
+      }).then(function(result) {
+          callback(result);
+      }).catch(function(err) {
+        callback(err)
+      });
+    },
+    delete: (params, callback) => {
+      Image.destroy({
+        where: {
+          userid: parseInt(params)
+        }
+      }).then(function(err, result) {
+        callback(err, result);
+      })
+    }
+  },
+  restaurant: {
+    post: (params, callback) => {
+      Image.create({
+        comment: params[1][0],
+        date: params[1][1],
+        imageurl: params[1][2],
+        userid: params[0]
```

Comment:
`userid` is a bit confusing of a variable name since it is actually the restaurant id if I understand correctly. Not a blocker but would file as tech debt.